src/main/webapp/WEB-INF/static/js/angular-tests/trialmanager/datasetService.spec.js

```diff
@@ -95,7 +95,7 @@ describe('Dataset Service', function () {
 			var mockData = {};
 
 			$httpBackend.whenGET('/bmsapi/crops/maize/studies/' + studyContext.studyId + '/datasets/' +
-				datasetId + '/instances')
```

Comment:
Shouldn't this be retained to `instances`?

Reply:
Done